calendar/data_mappings.bal

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// isolated function convertToEvent(map<json> sourceEventObject) returns Event|error {
+//     Event event = check sourceEventObject.cloneWithType(Event);
+//     event.downloadUrl = let var url = sourceEventObject["@microsoft.graph.downloadUrl"] in url is string ?
+//         url : EMPTY_STRING;
+//     return event;
+// }
```

Comment:
Shall we remove this, if this is not needed ?

Reply:
Removed the data_mapping.bal 
/Thanks